### PR TITLE
Feat(board): add the redux implementation, for Electron

### PR DESCRIPTION
**Pull request by YvesLongchamp**
Here, you can found all of the boards
## refactor: add actions and reducers

those are not working, for now. A fix is necessary

## fix(board): fix the actions and reducers

we still have to correct the config file

## refactor(board): change the actions, and reducers

change of the actions, to be usable for multiple boards
a change to do : add multiple boards for the board.tsx

## feat: add the board component
(a simplist version)

## feat(board): add the tests
for the component, the reducers, and actions

## refactor: change some code, to be better written

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,20 +4,38 @@
   "private": true,
   "dependencies": {
     "electron": "^1.7.9",
+    "enzyme": "^3.1.0",
```
Comment (Porquepix) [inline package.json]: All packages related to tests should be in dev part.

File: src/components/Board/DnDContextBoard.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { DragDropContext } from 'react-beautiful-dnd'
+
+import Board, { BoardProps } from './Board'
+
+const DnDContextBoard = (props2: BoardProps) => {
```
Comment (Porquepix) [inline src/components/Board/DnDContextBoard.tsx:6]: Why do we need a DnD ?

File: package.json (hunk 1 above)
Comment (YvesLongchamp) [reply package.json]: Will be fixed in the next commit.

File: src/components/Board/DnDContextBoard.tsx (hunk 2 above)
Comment (YvesLongchamp) [reply src/components/Board/DnDContextBoard.tsx:6]: Drag and drop seems good, if you want to move the boards on a list. Keep the important ones before, etc...
Seems better to try to implement it now, and remove it if we do not need it.

**Comment by YvesLongchamp** [comment]
@Porquepix Changes are done, please confirm everything is good.

**Review by Porquepix — APPROVED** [review]

**Pull request merged**